kiliautoml/models/__init__.py

```diff
@@ -0,0 +1,7 @@
+# flake8: noqa
```

Comment:
it's a bit drastic to ignore the whole file. We could just add some specific rules.

Reply:
I know, but for some reason flake8 ignores the #noqa directives when they are added in multiple import 🤷‍♂️. So I did not have any choice